config/routes.rb

```diff
@@ -1,12 +1,21 @@
 Rails.application.routes.draw do
+  # DFE-Digital/dfe-wizard auto-routing
+  concern :wizardable do |options = {}|
```

Comment:
I'd never come across [`concern`](https://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Concerns.html#method-i-concern) before, but this looks useful!